src/services/clientService.ts

```
@@ -1,13 +1,60 @@
+        const queryParams = Object.fromEntries(new URLSearchParams(queryString))
+
+        // Extract page and the rest of the path segments
+        const [_, page, ...rest] = urlPath.split('/')
+
+        const title = await getPageTitle(page, rest, queryParams)
+        const description = await getPageDescription(page, rest, queryParams);
+
+        htmlData = htmlData.replaceAll("____META_OG_TITLE____", title);
+        htmlData = htmlData.replaceAll("____META_DESCRIPTION____", description);
+
+        htmlData = htmlData.replaceAll("____META_OG_IMAGE_URL____", config.FRONT_END_URL + '/og/' + (page || "unit") + (!page && (queryParams.plot || queryParams.house) ? ('?' + new URLSearchParams(queryParams).toString()) : ''));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)